### PR TITLE
Spurious dragon 2

### PR DESCRIPTION
**Pull request by pirapira**
I considered recent changes on `master` and adjusted the Spurious Dragon branch.

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -233,6 +233,31 @@ \subsection{World State} \label{ch:state}
...
 \end{equation}
 
+An account state is empty when it has no code, zero nonce and zero balance:
+\begin{equation}
+\mathtt{\tiny EMPTY}(\boldsymbol{\sigma}, a)
+\quad\equiv\quad \boldsymbol{\sigma}[a]_c = \texttt{\small KEC}\big(()\big)
+\wedge \boldsymbol{\sigma}[a]_n = 0
+\wedge \boldsymbol{\sigma}[a]_b = 0
+\end{equation}
+Even callable precompiled contracts can have an empty account state.  This is because their account states do not usually contain the code describing its behavior.
```
Comment (pirapira) [inline Paper.tex]: @arkpar Does this line suffice as the clarification?

Comment (arkpar) [inline Paper.tex]: Quite so, thank you.

File: Paper.tex [hunk 2]
```diff
@@ -699,9 +725,12 @@ \section{Contract Creation} \label{ch:create}
 Code execution depletes gas, and gas may not go below zero, thus execution may exit before the code has come to a natural halting state. In this (and several other) exceptional cases we say an out-of-gas (OOG) exception has occurred: The evaluated state is defined as being the empty set, $\varnothing$, and the entire create operation should have no effect on the state, effectively leaving it as it was immediately prior to attempting the creation.
 
 If the initialization code completes successfully, a final contract-creation cost is paid, the code-deposit cost, $c$, proportional to the size of the created contract's code:
-\begin{equation}
-c \equiv G_{codedeposit} \times |\mathbf{o}|
-\end{equation}
+\begin{align}
+c \equiv \begin{cases}
+  2^{256} - 1 & \text{if} \quad |\mathbf{o}| \le 24576 \\
```
Comment (gavofyork) [inline Paper.tex]: would suggest `\varnothing`, then check for this (along with `g<c`) later on to determine whether it's reverted.

File: Paper.tex [hunk 3]
```diff
@@ -1419,7 +1456,7 @@ \section{Signing Transactions}\label{app:signing}
...
 \end{eqnarray}
 
-Where $p_u$ is the public key, assumed to be a byte array of size 64 (formed from the concatenation of two positive integers each $< 2^{256}$) and $p_r$ is the private key, a byte array of size 32 (or a single positive integer in the aforementioned range). It is assumed that $v$ is the `recovery id', a 1 byte value specifying the sign and finiteness of the curve point; this value is in the range of $[27, 30]$, however we declare the upper two possibilities, representing infinite values, invalid.
+Where $p_u$ is the public key, assumed to be a byte array of size 64 (formed from the concatenation of two positive integers each $< 2^{256}$) and $p_r$ is the private key, a byte array of size 32 (or a single positive integer in the aforementioned range). It is assumed that $v$ is either the `recovery id' or `chain id doubled plus 35 or 36'.  The recovery id is a 1 byte value specifying the sign and finiteness of the curve point; this value is in the range of $[27, 30]$, however we declare the upper two possibilities, representing infinite values, invalid.  The chain id is a constant natural number.
```
Comment (gavofyork) [inline Paper.tex]: "id" -> "identity"

**Comment by pirapira** [comment]
I'll split this PR EIP-wise.

**Pull request closed without merging**